src/Hotel.java

```diff
@@ -5,24 +5,29 @@
     private String name;
     private ArrayList<Room> rooms;
 
-    Hotel(String name, ArrayList<Room> rooms) {
+    Hotel(String name, ArrayList<Room> rooms)
```

Comment:
Add Javadoc to the constructor, the class and all methods except getter and setters and overriden java.lang.Object methods.
The JavaDoc should have a brief explanation about the class, constructor, method so people using/checking your code can have a basic understanding before starting to read the implementation.